### PR TITLE
chore: handle new target source (MAPCO-2696)

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: MergerService/MergerService.csproj [hunk 1]
```diff
@@ -7,13 +7,13 @@
     <RunWorkingDirectory>$(MSBuildProjectDirectory)</RunWorkingDirectory>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.14.0"/>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1"/>
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3"/>
-    <PackageReference Include="GeoJSON" Version="2.3.1"/>
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.14.0" />
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply MergerService/MergerService.csproj]: fixed

File: .idea/.idea.GpkgMerger.dir/.idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Default ignored files
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply .idea/.idea.GpkgMerger.dir/.idea/.gitignore]: fixed

**Review by asafMasa — APPROVED** [review]

File: MergerService/Utils/TaskUtils.cs [hunk 3]
```diff
@@ -33,6 +36,9 @@ public class TaskUtils : ITaskUtils
             // Construct Json serializer settings
             _jsonSerializerSettings = new JsonSerializerSettings();
             _jsonSerializerSettings.Converters.Add(new StringEnumConverter());
+            this._jsonSerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
```
Comment (shimoncohen) [inline MergerService/Utils/TaskUtils.cs:39]: Intresting.

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**